anime_downloader/extractors/animeonline360.py

```diff
@@ -0,0 +1,18 @@
+from anime_downloader.extractors.base_extractor import BaseExtractor
+from anime_downloader.sites import helpers
+
+class AnimeOnline360(BaseExtractor):
+    def _get_data(self):
+        try:
+            html = helpers.soupify(helpers.get(self.url))
+            soup = html.find('li', class_='dooplay_player_option')
+            anime_type = soup['data-type']
+            anime_id = soup['data-post']
+            referrer = helpers.get(f"https://animeonline360.me/wp-json/dooplayer/v1/post/{anime_id}?type={anime_type}&source=1").json()['embed_url']
+            stream_url = referrer.split('?source=')[1].split('&')[0].replace('%2F', '/').replace('%3A', ':')

```

Comment:
Would personally use regex here, also the way you do .replace() can lead to future issues. Are they necessary? can you use something like urllib

Reply:
i can do this instead: https://stackoverflow.com/questions/16566069/url-decode-utf-8-in-python

Reply:
it was just a lazy and fast solution i had, i will use the url decode